torch/_inductor/ir.py

```diff
@@ -690,6 +690,8 @@ def create(
             rtypes_to_inits = {
                 "sum": 0,
                 "prod": 1,
+                "any": 0,
+                "all": 0,
```

Comment:
there's no "all" reduction type (and if there was, it would have `1` init value, try it).